b2sdk/session.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         :param api_config
         """
 
-        self.raw_api = B2RawHTTPApi(B2Http(api_config))
+        self.raw_api = (api_config.raw_api_class or B2RawHTTPApi)(B2Http(api_config))
```

Comment:
I don't think we should be setting a default here. Config must have it.

Reply:
Yes. I didn't want to put it explicitly as a default value for `B2HttpApiConfig.__init__`, because `B2RawHttpApi` is part of the internal API, but it should be an evaluated default there. As some say: "there is a better way"